css/dashboard.css

```diff
@@ -136,13 +150,173 @@ main .card {
   line-height: 19px;
   color: #222222;
   font-weight: 700;
+  padding: 0.75rem;
+}
+
+.thead_light th {
+  background-color: #f8f8f8;
+  border: none;
+  color: #222;
+  font-size: 14px;
+  line-height: 16px;
+  padding: 0.75rem;
 }
 
 .table-box {
-  /* height: 432px; */
   background: #ffffff;
   box-shadow: 0px 8px 40px rgba(0, 0, 0, 0.05);
   border-radius: 16px;
   padding: 1rem;
   margin-right: 0.5rem;
 }
+
+tbody tr:not(:first-child) {
+  border-top: 1px solid #eaedf3;
+}
+
+.table__row td {
+  padding: 1.5rem 0.25rem;
+}
+
+.bd-example {
+  position: relative;
+  padding: 1rem;
+  margin: 1rem -15px 0;
+  border-width: 0.2rem 0 0;
+}
+
+.result__detail {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 0;
+  padding: 1.5rem 0;
+}
+
+.result__details .result__detail:not(:first-child) {
+  border-top: 1px solid #eaedf3;
+}
+
+.result_name {
+  width: 100%;
+}
+
+.result_name p,
+.result_score p {
+  padding: 0;
+  margin-bottom: 0.25rem;
+}
+
+.result_score p {
+  text-align: right;
+  font-size: 14px;
+  line-height: 16px;
+  font-weight: 600;
+  color: #5fd172;
+}
+
+.result_name p:nth-child(1),
+.mini__card__text p:nth-child(1) {
+  /* Black */
+  color: #222222;
+  font-weight: 600;
+  font-size: 12px;
+}
+
+.result_name p:nth-child(2),
+.mini__card__text p:nth-child(2) {
+  font-size: 12px;
+  opacity: 0.4;
+}
+
+.result_abbr {
+  background: #eff2fe;
+  border-radius: 8px;
+  height: 3rem;
+  font-size: 18px;
+  line-height: 21px;
+  color: #2a5699;
+  width: 5.2rem;
+  text-align: center;
+  align-items: center;
+  display: flex;
+  justify-content: center;
+  margin-right: 1rem;
+}
+
+.result_progress {
+  width: 100%;
+  height: 0.4rem;
+  font-weight: bold;
+  border-radius: 8px;
+  background-color: #f0f0f0;
+}
+
+.result_progress::-webkit-progress-bar {
+  background-color: #5fd172;
+  border-radius: 8px;
+}
+
+.result_progress::-webkit-progress-value {
+  border-radius: 2px;
+  background-color: #f0f0f0;
+}
+
+.result_progress::-moz-progress-bar {
+  background-color: #5fd172;
+  border-radius: 8px;
+}
+
+.box__grid {
+  display: grid;
+  gap: 1rem;
+  grid-template-columns: repeat(2, 1fr);
+}
+
+.card__custom {
+  border-radius: 16px;
+  flex-direction: row;
+  max-height: 7rem;
+  position: relative;
+}
+
+.mini__card__text {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.mini__card__text p {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected selector ".mini__card__text p" to come before selector ".mini__card__text p:nth-child(2)" (no-descending-specificity)](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5469161)